Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:10-alpine
+FROM node:14.15.4
```

Comment:
@sulthan-ahmed It would be good to chat to you about this. I saw that support for node 10 is being stopped in the next few months. It all works locally with the updated node version but I'm not sure of the wider implications of updating this.

Reply:
imho upgrades should be a separate PR 🙂

Reply:
@joehod

Reply:
@MaxOSullivan  is making a separate MR to update the node version